lib/src/future.dart

```diff
@@ -0,0 +1,9 @@
+import 'dart:async';
```

Comment:
Can you use a `part of`  instead of using an import statement in this file and import async in the `flutter_extension.dart` file.
You can see the other extension files for the same.